watchtower/conf_experimental.go

```diff
@@ -0,0 +1,65 @@
+// +build experimental
+
+package watchtower
+
+import (
+	"time"
+
+	"github.com/lightningnetwork/lnd/lncfg"
+)
+
+// Conf specifies the watchtower options that can be configured from the command
+// line or configuration file.
+type Conf struct {
+	RawListeners []string `long:"listen" description:"Add interfaces/ports to listen for peer connections"`
+
+	ReadTimeout time.Duration `long:"readtimeout" description:"Duration the watchtower server will wait for messages to be received before hanging up on clients"`
+
+	WriteTimeout time.Duration `long:"writetimeout" description:"Duration the watchtower server will wait for messages to be written before hanging up on client connections"`
+}
+
+// Apply completes the passed Config struct by applying any parsed Conf options.
```

Comment:
Looks to be over 80 characters at a glance.

Reply:
it's exactly 80 😉